policyengine_us/parameters/gov/states/mi/tax/income/credits/eitc/max_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Michigan provides this maximum amount of tax credits under the state EITC.
+metadata:
+  unit: currency-USD
+  label: Michigan EITC max amount
+  reference:
+    - title: Government Michigan - Earned Income Tax Credit for Working Families
+      href: https://www.michigan.gov/taxes/iit/eitc
+values:
+  2023-01-01: 2_080
```

Comment:
Can you check the effective date against a tax form? I suspect this was the cap for 2022, but the tentative expansion makes it unclear.